modules/compute/eks/docs/eks-isolated/eks-isolated-eksmanifest.yaml

```diff
@@ -0,0 +1,117 @@
+name: eks
+path: modules/core/eks/
+dataFiles:
```

Comment:
The local reference breaks the gitpath functionality, doesn't it?  How would this module deploy from a gitpath definition?